src/Controller/SecurityController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function login(AuthenticationUtils $authenticationUtils): Response
+    {
+        // get the login error if there is one
+        $error = $authenticationUtils->getLastAuthenticationError();
```

Comment:
I find the comment helpful for now, so I will just leave it there.

Reply:
The method explains exactly the same thing, the comment is redundant?

Reply:
I know but this "if there is one" helps me to realize that there doesn't have to be any error... Otherwise I would wonder what that error is... Just for now, in the beginning, I would keep it. I think it is not crucial anyway.